From ad199c76d9b61fb541127864e169aab3bf1a45b3 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Wed, 11 Apr 2018 14:29:07 +0200
Subject: [PATCH 1/2] activitymanager: Add service file for systemd

In order to get rid of webos-initscripts, add the required files to the components.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 files/systemd/activitymanager.service | 29 +++++++++++++++++++++++++++
 1 file changed, 29 insertions(+)
 create mode 100644 files/systemd/activitymanager.service

diff --git a/files/systemd/activitymanager.service b/files/systemd/activitymanager.service
new file mode 100644
index 0000000..04c231d
--- /dev/null
+++ b/files/systemd/activitymanager.service
@@ -0,0 +1,29 @@
+# Copyright (c) 2017-2018 LG Electronics, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Open webOS component to manage all running activities.
+
+[Unit]
+Description=webos - "%n"
+Requires=ls-hubd.service configurator-db8.service
+After=ls-hubd.service configurator-db8.service
+
+[Service]
+Type=notify
+OOMScoreAdjust=-500
+EnvironmentFile=-/var/systemd/system/env/activitymanager.env
+ExecStart=/usr/sbin/activitymanager -c {\"log\":{\"appender\":{\"type\":\"syslog\"},\"levels\":{\"default\":\"notice\"}}}
+Restart=on-failure
-- 
2.37.3.windows.1

